src/services/mongo.js

```
@@ -14,8 +14,16 @@ const connectToDB = () => {
     useUnifiedTopology: true,
   };
 
+  if (typeof process.env.MONGO_URI === 'undefined') {
+    logger.fatal('Connection unsuccessful: the URI was not provided');
+  }
```

```suggestion
  if (typeof process.env.MONGO_URI === 'undefined') {
    logger.fatal('Connection unsuccessful: the URI was not provided');
    return;
  }
```

---

We should throw an error so that the caller can handle it and know it failed

---

> We should throw an error so that the caller can handle it and know it failed

True, updated my comment